deployment-k8s/aws-s3-k8s/example-runner/deployServicesToK8s.sh

```diff
@@ -0,0 +1,88 @@
+#! /usr/bin/env bash
+# Copyright 2018-2021 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+helpFunction() {
+   echo ""
+   echo "Usage: $(basename "$0") [OPTIONS]"
+   echo -e "\t-n(amespace)                The name for the K8s namespace to deploy to"
+   echo -e "\t-r(epository)               The URL for the (ECR) repository"
+   echo -e "\t-h(ostname)                 The URL for the (ELB) hostname of the cluster deployment"
+   echo -e "\t-d(atastore)                The URL for the (EFS) aws volume handle used as a data-store"
+   echo -e "\t-c(lasspathjars)            The URL for the (EFS) aws volume handle used for storing classpath JARs"
+   echo -e "\t-t(raefik)                  The boolean true/false value for installing traefik"
+   echo -e "\t-P(refix)                   The topic prefix so we generate unique topic names"
+   echo -e "\t-R(ole)                     Aws iam role name"
```

Comment:
What is this role for? What permissions does it need?
S3 READ permissions i presume, which is get/list on object/bucket mostly